_search-plugins/star-tree-index.md

```diff
@@ -425,10 +472,6 @@ POST /sales/_search
 ```
 {% include copy-curl.html %}
 
-#### Nested Aggregations
-
-Bucket aggregations supported with star-tree index search can be nested and resolved using star-tree. [See more]({{site.url}}{{site.baseurl}}/aggregations/#nested-aggregations).
-
-## Using queries without a star-tree index
+#### Nested aggregations
 
-Set the `indices.composite_index.star_tree.enabled` setting to `false` to run queries without using a star-tree index.
```

Comment:
@kolchfa-aws We should not remove this section. We need to still give the user an option to disable index search via star-tree.

Reply:
Done

Reply:
Can we kindly keep this a separate section. Today with feature flag removal, and cluster setting pointing to true, its by default enabled. So its not coming out clearly that this can be done. Also we have added index level setting so adding the info in separate section will be useful.

Reply:
Done.